src/Tablet/Dashboard/Tile.module.css

```diff
@@ -0,0 +1,16 @@
+.card {
+    display: flex!important;
+    font-weight: bold;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 4px!important;
```

Comment:
or simpler: `margin: 4px`
More precise: `margin: 0.25rem`
Even better for this app: `margin: var(--rd-spc-xxs)` (see https://github.com/RoboDomo/robodomo-client/blob/master/src/theme.css#L22)